consensus/view_change.go

```diff
@@ -468,22 +287,18 @@ func (consensus *Consensus) onViewChange(msg *msg_pb.Message) {
 					block.NumberU64(),
 					block.Header().ViewID().Uint64(),
 				); err != nil {
-					consensus.getLogger().Warn().Msg("submit vote on viewchange commit failed")
+					consensus.getLogger().Warn().Err(err).Msg("submit vote on viewchange commit failed")
 					return
 				}
 
 			}
 		}
 
-		consensus.SetViewChangingID(recvMsg.ViewID)
+		consensus.SetViewIDs(recvMsg.ViewID)
```

Comment:
Should not update block view ID, because no new block is confirmed. Only view change reached quorum, doesn't mean the rest will succeed and a new consensus can be reached yet.